src/ArcGISEarth.AutoAPI.Examples/MainWindowViewModel.cs

```diff
@@ -211,6 +223,10 @@ public string PrettyJson(string jsonString)
         private const string GETLAYER_EXAMPLE = "311b7317-94f8-4f80-89f2-0e3ca5e77d28";
         private const string REMOVELAYER_EXAMPLE = "311b7317-94f8-4f80-89f2-0e3ca5e77d28";
         private const string REMOVELAYERS_EXAMPLE = "operationalLayers";
+        private const string ADDGRAPHIC_EXAMPLE = "{ \"geometry\": {\"type\": \"point\", \"x\": -100, \"y\": 40 }, \"symbol\": { \"type\": \"picture-marker\", \"url\": \"https://static.arcgis.com/images/Symbols/Shapes/BlackStarLargeB.png\", \"width\": \"64px\", \"height\": \"64px\", \"xoffset\": \"10px\", \"yoffset\": \"10px\"}}";
```

Comment:
Do you need to set a graphic id to the json to ensure that you can call get graphic successfully?

Reply:
I think it's not necessary as there is an error response if call failed. The example is also the same as the JS example.

Reply:
Sounds good.